src/helpers/provider.ts

```diff
@@ -57,3 +56,22 @@ export function walkProviders(
   }
   throw new Error(`Unable to detect provider.`)
 }
+
+export function setSlug(
+  slugArg: string,
```

Comment:
```suggestion
  slugArg: string || '',
```

It may not be immediately clear _why_ it would be necessary to even call this function if `slugArg` is typed as a string. Specifying that it must at least be an empty string helps clear that up.

Essentially, by moving the type from `string | undefined` to `string`, we lose the static typing of that "empty" state. We can remediate that by specifying, although somewhat redundantly, that an empty string is a valid type (in the context of the functions currently under review).

Reply:
`string | string` does not change anything syntactically nor from a documentation perspective. `string | ''` conveys that an empty string is quite literally a valid and expected shape. Since we use it in lieu of `undefined` as we previously used for the "empty" value, it presents documentation details although it is syntactically redundant.

Reply:
` slug: string || ''` fails linting.